app/javascript/components/venue/VenueNew.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+import Grid from '@material-ui/core/Grid';
+import { FormTextField } from '../../ui/TextFileds';
+import { FormButton } from '../../ui/Buttons';
+
+export default class VenueNew extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      venue: { address: '', description: '', people_capacity: '' }
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange = (venue) => {
+    var prevState = {...this.state};
+    var updatedVenue = {...this.state.venue};
+    updatedVenue[venue.target.name] = venue.target.value;
+    this.setState({ ...prevState, venue: updatedVenue });
+  }
+
+  handleSubmit(venue) {
+    venue.preventDefault();
```

Comment:
this line is confusing. preventDefault is method for js event, better to name this variable event

Reply:
done